### ISSUE TITLE
ZipShortest overloads missing return value documentation 

### ISSUE THREAD

**Issue by atifaziz** [morelinq/MoreLINQ#149]
Warnings running `builddocs.cmd` on cc6932f74cfcc7476bcc99db10f450bfdcb010c0:

```
BuildAssembler : warning : ShowMissingComponent: [M:MoreLinq.MoreEnumerable.ZipShortest``5(System.Collections.Generic.IEnumerable{``0},System.Collections.Generic.IEnumerable{``1},System.Collections.Generic.IEnumerable{``2},System.Collections.Generic.IEnumerable{``3},System.Func{``0,``1,``2,``3,``4})] Missing <returns> documentation [A:\MoreLINQ\main\docs\api\Working\BuildReferenceTopics.proj]
BuildAssembler : warning : ShowMissingComponent: [M:MoreLinq.MoreEnumerable.ZipShortest``4(System.Collections.Generic.IEnumerable{``0},System.Collections.Generic.IEnumerable{``1},System.Collections.Generic.IEnumerable{``2},System.Func{``0,``1,``2,``3})] Missing <returns> documentation [A:\MoreLINQ\main\docs\api\Working\BuildReferenceTopics.proj]
BuildAssembler : warning : ShowMissingComponent: [M:MoreLinq.MoreEnumerable.ZipShortest``3(System.Collections.Generic.IEnumerable{``0},System.Collections.Generic.IEnumerable{``1},System.Func{``0,``1,``2})] Missing <returns> documentation [A:\MoreLINQ\main\docs\api\Working\BuildReferenceTopics.proj]
```

**Comment by VladimirRybalko** [comment]
@atifaziz I've created a request to fix it.

**Comment by atifaziz** [comment]
Closed by 09aa5adcd4a10f693528f5044dd252f893fe962d from PR #182.
